docs/source/index.rst

```diff
@@ -4,48 +4,95 @@ Performance, Portability, and Productivity Analysis Library
 .. toctree::
    :maxdepth: 1
    :hidden:
+   :caption: Introduction
 
-   introduction
+   Getting Started <self>
+   analysis
    data
+
+.. toctree::
+   :maxdepth: 1
+   :hidden:
+   :caption: Tutorial
+
    examples/index
    case-studies/index
+
+.. toctree::
+   :maxdepth: 1
+   :hidden:
+   :caption: Reference
+
    p3
+
+.. toctree::
+   :maxdepth: 1
+   :hidden:
+   :caption: Contributing
+
+   How to Contribute <https://github.com/intel/p3-analysis-library/blob/main/CONTRIBUTING.md>
+   GitHub <https://github.com/intel/p3-analysis-library>
    notices-and-disclaimers
 
 The Performance, Portability, and Productivity Analysis Library (P3 Analysis
 Library) enables a simpler workflow for the collection and interpretation of P3
 data.
 
+- Compute :doc:`metrics <p3.metrics>` like "performance portability" and
+  "code divergence" to quantify the trade-offs between performance, portability
+  and productivity made by applications targeting multiple platforms.
+
+- :doc:`Plot <p3.plot>` cascades and navigation charts to visualize and gain
+  deeper insight into the "performance portability" and "code divergence"
+  scores achieved by different applications.
+
+
+Installation
+############
+
+The latest release of the P3 Analysis Library is version 0.1.0-alpha. To
+download and install this release, run the following::
+
+    $ git clone --branch v0.1.0-alpha https://github.com/intel/p3-analysis-library.git
+    $ cd p3-analysis-library
+    $ pip install .
```

Comment:
Just to note that there is a p3 package within pip.

I have had people run `pip install p3` and then ask why cascade doesn't exist etc.

Even if you realise, doing `pip install .` after fails because the p3 version is newer than the _p3_ version here.

While this library will likely not be added to pip directly, there still might be some consideration for renaming the library to "p3l" or "p3lib", etc

Reply:
Oof, okay. If people are actually making this mistake and it's no longer hypothetical, we should fix it.  I think we should address it separately, though, so I've created a new issue to track it: https://github.com/intel/p3-analysis-library/issues/48.